lib/v2/scitation/journal.js

```diff
@@ -0,0 +1,69 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const config = require('@/config').value;
+const path = require('path');
+const { art } = require('@/utils/render');
+
+const { CookieJar } = require('tough-cookie');
+const cookieJar = new CookieJar();
+
+module.exports = async (ctx) => {
+    const pub = ctx.params.pub;
+    const jrn = ctx.params.jrn;
+    const host = 'https://'.concat(pub, '.scitation.org');
+    const jrnlUrl = host.concat('/toc/', jrn, '/current?size=all');
```

Comment:
```suggestion
    const host = `https://${pub}.scitation.org`;
    const jrnlUrl = `${host}/toc/${jrn}/current?size=all`;
```
Please use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) or [+](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Addition) instead of [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#performance)